lstm_electricity_consumption/lstm_electricity_consumption.cpp

```diff
@@ -93,33 +93,30 @@ void SaveResults(const string& filename,
                  data::MinMaxScaler& scale,
                  const arma::cube& testX)
 {
-  arma::mat flatDataAndPreds = testX.slice(testX.n_slices - 1);
-
-  // The prediction result is the energy consumption for the next hour and comes
-  // from the last slice of the prediction.
-  flatDataAndPreds.rows(flatDataAndPreds.n_rows - 1,
-                        flatDataAndPreds.n_rows - 1) =
-      predictions.slice(predictions.n_slices - 1);
-
-  scale.InverseTransform(flatDataAndPreds, flatDataAndPreds);
-  // We need to remove the last column because it was not used for training
-  // (there is no next hour to predict).
-  flatDataAndPreds.shed_col(flatDataAndPreds.n_cols - 1);
-
-  // Save the data to file. The last columns are the predictions; the preceding
-  // column is the data used to generate those predictions.
-  data::Save(filename, flatDataAndPreds);
-
-  // Print the output to screen.
-  // NOTE: we do not have the last data point in the input for the prediction
-  // because we did not use it for the training, therefore the prediction result
-  // will be for the hour before. In your own application you may of course load
-  // any dataset for prediction.
-  cout << "The predicted energy consumption for the next hour is : " << endl;
-  cout << " "
-       << flatDataAndPreds(flatDataAndPreds.n_rows - 1,
-                           flatDataAndPreds.n_cols - 1)
-       << endl;
+  arma::mat flatDataAndPreds = testX.slice(testX.n_slices - 1);
+  scale.InverseTransform(flatDataAndPreds, flatDataAndPreds);
+
+  //The prediction result is the energy consumption for the next hour and 

```

Comment:
```suggestion
  // The prediction result is the energy consumption for the next hour and 
```
I can't seem to do multiline suggestions on a phone... sorry that these are split up...